App/Containers/ChartScreen/ChartScreen.js

```diff
@@ -8,8 +8,8 @@ export default class ChartScreen extends React.Component {
   render() {
     const labels = lastNDays(7);
     return (
-      <View style={[Helpers.fillRowCenter, styles.container]}>
-        <Chart title="Past savings" data={[-10, -15, 40, 19, 32, 15, 52]} xLabels={labels}/>
```

Comment:
Why get rid of the title?

Reply:
I think I took it out because its not in the mockup, but I just added it back in.